app/templates/codelists-management/codelist.hbs

```diff
@@ -0,0 +1,110 @@
+{{page-title (t "codelist.details")}}
+
+<BreadcrumbsItem as |linkClass|>
+  <LinkTo
+    @route="codelists-management.codelist"
+    @model={{@model.codelist.id}}
+    class={{linkClass}}
+  >
+    {{@model.codelist.readableLabel}}
+  </LinkTo>
+</BreadcrumbsItem>
+
+<div class="au-o-grid au-o-grid--flush au-o-grid--fixed">
+  <div
+    class="au-o-grid__item {{
+    if
+      this.showSidebar
+      "au-u-3-5@medium au-u-visible-from@medium"
+      "au-u-1-1@medium"
+    }}"
+  >
+    <div class="au-c-body-container au-c-body-container--scroll">
+      <AuToolbar @size="large">
+        <AuToolbarGroup>
+          <AuHeading @skin="2">
+            {{t "codelist.details"}}
+          </AuHeading>
+        </AuToolbarGroup>
+      </AuToolbar>
+      <AuToolbar @size="medium">
+        <AuToolbarGroup class="au-c-toolbar__group--row">
+          <ul class="au-o-grid">
+            <li class="au-o-grid__item au-u-1-2 au-u-1-6@medium">
+              <AuLabel>
+                {{t "codelist.attr.label"}}
+              </AuLabel>
+              <p>
+                {{@model.codelist.readableLabel}}
+              </p>
+            </li>
+            <li class="au-o-grid__item au-u-1-2 au-u-1-3@medium">
+              <AuLabel>
+                {{t "codelist.attr.values"}}
+              </AuLabel>
+              <p>
+                <ul>
+                  {{#each @model.codelist.codeListOptions as |option|}}
+                    <li>
+                      {{option.label}}
+                    </li>
+                  {{/each}}
+                </ul>
+              </p>           
+            </li>
+          </ul>
+        </AuToolbarGroup>
+      </AuToolbar>
+      <AuToolbar @border="bottom" @size="large">
+        <AuToolbarGroup>
+          <AuLink
+            @skin="button"
+            @route="codelists-management.edit"
+            @model={{@model.codelist.id}}
+          >
+            {{t "codelist.crud.edit"}}
+          </AuLink>
+          <AuButton
+            @skin="secondary"
+            @alert={{true}}
+            {{on "click" this.toggleIsOpen}}
+          >
+            {{t "codelist.crud.delete"}}
+          </AuButton>
+          <AuModal
+            @modalTitle={{t "utility.confirmation.title"}}
+            @modalOpen={{this.isOpen}}
+            @closeModal={{this.toggleIsOpen}} as |Modal|
+          >
+            <Modal.Body>
+              <p>
+                {{t "utility.confirmation.body"}}
+              </p>
+            </Modal.Body>
+            <Modal.Footer>
+              <AuButton
+                @alert={{true}}
+                {{on
+                  "click"
+                  (fn this.removeCodelist @model.codelist)
```

Comment:
I don't think the codelist needs to be passed in since the controller can already access it through the model?